warehouse/utils/utils.go

```diff
@@ -652,7 +652,10 @@ func GetLoadFileType(wh string) string {
 	case "BQ":
 		return LOAD_FILE_TYPE_JSON
 	case "RS":
-		return LOAD_FILE_TYPE_PARQUET
+		if config.GetEnvAsBool("WAREHOUSE_RS_USE_PARQUET_LOAD_FILES", false) {
```

Comment:
@dhawal-rudder  actually, lets move to use hot reloadable variable instead of ENV one